AWSCore/STS/Info.plist

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why is this created now?

Reply:
+ @phani-srikar, for visibility

Reply:
moving this to draft, so to remember not to check this in until we are ready